async port scanner.py

```diff
@@ -0,0 +1,48 @@
+"""
+Async port scanner.
+"""
+
+import time
+import asyncio
+
+
+async def connect_socket(ip: str, port: int):
+    """
+    Connect to given IP and port
+
+    :param ip: The ip for the socket to connect.
+    :param port: The port for the socket to connect.
+    """
+    try:
+        await asyncio.open_connection(ip, port)
+        print(f"port {port} found!")
+    except WindowsError:
+        print("Connection failed!")
```

Comment:
As I said. This is good, but I know that by knowing what didn't fail. You can also not print this.

Reply:
If you don't print this then the `except` block is empty and you can remove it. Use `suppress` package in Python for more elegant way.